src/lib/ModelConfig.cc

```diff
@@ -621,6 +621,16 @@ void ModelConfig::validate() {
   }
   subFields.clear();
 
+  // Environment-Variables
+  root = "Environment-Variables";
+  size_t numEnvVars = configFile_[root].size();
+  for (size_t i = 0; i < numEnvVars; i++) {
+    char envVarNum[50];
+    snprintf(envVarNum, 50, "Environment-Variable %zu ", i);
```

Comment:
Why couldn't you do: 
`std::string envVarStr = "Environment-Variable " + std::to_string(i);`

Reply:
updated